cypress/support/api/requests/Checkout.js

```diff
@@ -72,6 +73,8 @@ export function createCheckout({
       }
     }
   }`;
+
+  cy.log(mutation);
```

Comment:
Remove it.

Reply:
checkout shouldn't be changed under this task, changes removed